src/collections/_documentation/performance/alerts-notifications/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Alerts & Notifications
+sidebar_order: 0
+---
+
+{% capture __alert_content -%}
+Sentry's Performance features are currently in beta. For more details about access to these features, feel free to reach out at [https://sentry.io/for/performance/](https://sentry.io/for/performance/).
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Sentry notifications are broadly categorized into alerts and non-alert notifications. Alerts can be sent to many supported [LINK: integrations]. Non-alert notifications only go to email.
+
+## Alerts
+
+Alerts are configured per-project and are of two types: Metric Alerts and Issue Alerts.
```

Comment:
"are of two types" seems kind of stiff. Perhaps "Sentry provides both Metric Alerts and Issue Alerts. You define either type of alert per project."